src/main/java/eu/deltasource/internship/hotelmanagementsystem/hotel/service/domain/commodities/Toilet.java

```diff
@@ -30,4 +37,14 @@ public void setHasWaterSavingTechnology(boolean hasWaterSavingTechnology) {
     public void setColor(String color) {
```

Comment:
There aren't much use cases in which the the toilet changes color... well sometimes they get brown, but that's a normal use case. Let's remove this method and have the toilet color set once from the constructor.